Add GARDEN pages

becky-gilbert opened the pull request:
Fixes #1210 
Fixes #1211 
Fixes #1212 
Fixes #1213 

This PR adds the GARDEN pages.
- childrenhelpingscience.com/garden (main/home page)
- childrenhelpingscience.com/garden/about (what is project GARDEN?)
- childrenhelpingscience.com/garden/participate (how to participate)
- childrenhelpingscience.com/garden/scientists (meet the scientists)

Notes:
- All pages have the following navigation tabs above the main content:
  - "Project GARDEN" -> /garden
  - "About" -> /garden/about
  - "How to Join" -> /garden/participate
  - "Start Now" -> /studies/garden (existing lab page)
  - "The Scientists" -> /garden/scientists
- All pages are mobile-friendly
- While working on the GARDEN Scientists page, I put the boilerplate HTML for staff/scientist/student profiles into a template tag and took the opportunity to refactor the main CHS Scientists page - see [next comment](https://github.com/lookit/lookit-api/pull/1221#issuecomment-1550396539).

## Main page

Desktop/laptop:

![Screenshot 2023-05-16 at 2 07 42 PM](https://github.com/lookit/lookit-api/assets/9041788/93315ecf-3dd1-4508-be77-675241019401)
![Screenshot 2023-05-16 at 2 07 52 PM](https://github.com/lookit/lookit-api/assets/9041788/42565271-847e-4bfa-b6e4-1a2af9e86555)

Mobile:

![Screenshot 2023-05-16 at 2 10 30 PM](https://github.com/lookit/lookit-api/assets/9041788/220965f2-bf5e-4a69-a476-138f118d3236)

## About

Desktop/laptop:

![Screenshot 2023-05-16 at 2 08 02 PM](https://github.com/lookit/lookit-api/assets/9041788/62f6750d-d519-4db7-a20d-89f2fbf64a5c)
![Screenshot 2023-05-16 at 2 08 11 PM](https://github.com/lookit/lookit-api/assets/9041788/0bbc07eb-c901-4eb6-ae16-03d49242d1f8)
![Screenshot 2023-05-16 at 2 08 17 PM](https://github.com/lookit/lookit-api/assets/9041788/ab296927-8a1d-4c01-959c-bda3935e1418)

Mobile:

![Screenshot 2023-05-16 at 2 10 59 PM](https://github.com/lookit/lookit-api/assets/9041788/671ea913-d4c6-44c5-8194-334e2c0f0c31)

## Participate

Desktop/laptop:

![Screenshot 2023-05-16 at 2 08 28 PM](https://github.com/lookit/lookit-api/assets/9041788/ea50bdc2-06ce-4066-9cce-a612b665a138)
![Screenshot 2023-05-16 at 2 08 47 PM](https://github.com/lookit/lookit-api/assets/9041788/625b068b-2446-4168-931c-f2cefdd0f3d4)

Mobile:

![Screenshot 2023-05-16 at 2 11 24 PM](https://github.com/lookit/lookit-api/assets/9041788/cfa05497-812e-46e6-a1eb-aef8d296d651)
![Screenshot 2023-05-16 at 2 11 31 PM](https://github.com/lookit/lookit-api/assets/9041788/b2281c7c-fac3-46dd-a56f-675f5f86b6d1)

## Scientists

Desktop/laptop:

![Screenshot 2023-05-16 at 2 09 00 PM](https://github.com/lookit/lookit-api/assets/9041788/b2c8f3c4-b1ca-4ebd-8bee-5fca10456890)
![Screenshot 2023-05-16 at 2 09 10 PM](https://github.com/lookit/lookit-api/assets/9041788/09550434-ac03-45bb-8171-c404def09aa8)
![Screenshot 2023-05-16 at 2 09 18 PM](https://github.com/lookit/lookit-api/assets/9041788/b96be9e1-780b-46cb-bfc8-d44e26206585)
![Screenshot 2023-05-16 at 2 09 29 PM](https://github.com/lookit/lookit-api/assets/9041788/575f4a47-20b5-425f-9a94-3474f7de09b8)
![Screenshot 2023-05-16 at 2 09 34 PM](https://github.com/lookit/lookit-api/assets/9041788/3fa30886-70dc-4bb6-8602-305d4c642d9f)

Mobile:

![Screenshot 2023-05-16 at 2 10 08 PM](https://github.com/lookit/lookit-api/assets/9041788/5b3da03b-1d41-4b5a-9f63-9acccd0b07b2)

becky-gilbert commented:
Because I moved the staff profile HTML into a template tag for the GARDEN scientists page, I refactored the main CHS scientists page using this same template tag. The page looks basically the same on desktop/laptop but is now more mobile-friendly.

I also subbed in the new photos for people who are both pages (as requested [here](https://github.com/lookit/lookit-api/pull/1208#pullrequestreview-1428857971)).

Here are the specific changes:
- Candice Mills' photo replaced with the one used on the GARDEN sites
- Julian Jara-Ettinger's bio now matches the one used on the GARDEN site (NB: I changed this because his position/title differed between the two pages, so I assumed the older CHS one was out of date)
- Mark Sheskin's bio now matches the one used on the GARDEN site (as requested [here](https://github.com/lookit/lookit-api/pull/1208#issuecomment-1544639526))
- Image files are now stored in the static images directory (see #1102) and follow the same naming convention

Note that the "Affiliated Universities and Researchers" section is NOT mobile-friendly (I was trying to avoid scope creep!). But I can make that change too if you want @mekline?

## CHS Scientists

**Desktop/laptop**

![Screenshot 2023-05-16 at 2 25 02 PM](https://github.com/lookit/lookit-api/assets/9041788/64066079-dfba-4517-a248-fdf53f46021c)
![Screenshot 2023-05-16 at 2 25 14 PM](https://github.com/lookit/lookit-api/assets/9041788/ae8ff9ab-37d3-416e-8630-a045f9da4263)
![Screenshot 2023-05-16 at 2 25 26 PM](https://github.com/lookit/lookit-api/assets/9041788/2c835d2f-23e3-43b6-b1db-c45e7c41f850)

**Mobile**

![Screenshot 2023-05-16 at 2 26 12 PM](https://github.com/lookit/lookit-api/assets/9041788/744b6e9c-1d4c-4a99-93fb-66e9a427e949)
![Screenshot 2023-05-16 at 2 26 24 PM](https://github.com/lookit/lookit-api/assets/9041788/d077850b-3d0d-4a6f-9c5c-52036a28c96e)
![Screenshot 2023-05-16 at 2 26 39 PM](https://github.com/lookit/lookit-api/assets/9041788/127876c0-2af5-49e0-889c-e001b4ccfd12)
![Screenshot 2023-05-16 at 2 26 51 PM](https://github.com/lookit/lookit-api/assets/9041788/23c94e5d-6f0e-4cb1-8a19-229fc21478f7)
![Screenshot 2023-05-16 at 2 27 08 PM](https://github.com/lookit/lookit-api/assets/9041788/c33b98e9-1ae4-4a67-9dd0-f38d0e6ec853)

becky-gilbert commented on web/templates/web/garden/_navigation.html:
```diff
@@ -0,0 +1,15 @@
+            {% nav_link request 'web:garden-home' 'Project GARDEN' list=True %}
+            {% nav_link request 'web:garden-about' 'About' list=True %}
+            {% nav_link request 'web:garden-participate' 'How to Join' list=True %}
+            <li class="nav-item">
+                <a class="nav-link navbar-link link-secondary text-center px-3"
+                   href="{{ base_url }}{{ garden_url }}">Start Now</a>
+            </li>
```
@okaycj do you know if it's possible to pass the link to the GARDEN lab site (`/studies/garden`) into the `nav_link` template tag here? I tried but couldn't figure it out :frowning_face:

okaycj replied:
You have an example of what I would try down below. 
```
{% url "web:lab-studies-list" lab_slug="garden" as garden_url %}
```

becky-gilbert replied:
Ah, so I forgot to mention that I did try exactly what you suggest here, with the nav link like this:
```
{% nav_link request garden_url 'Start Now' list=True %}
```
But that gave me this error when trying to load any page that uses the garden navigation: 
> Reverse for '/studies/garden/' not found. '/studies/garden/' is not a valid view function or pattern name.

The reason is that the `url` template tag returns the absolute path, i.e. `/studies/garden`, but the `nav_link` template tag expects the url argument to be in the form of a name for reverse lookup.

I just now realized that instead of trying to get the GARDEN studies page in the right form for reverse lookup, I could just edit the `nav_link` template tag so that it allows the option of receiving an absolute path. Does that seem like a reasonable solution to you?

ianchandlercampbell commented:
Ian Feedback:

First of all this all look amazing! I'm quite excited to see this subsite when it goes live. I have three suggestions:

1. **Text Size**: I like the normal text scaling/size on mobile but it looks very small on the desktop site. Could we make that font bigger so it is more readable?

2. **Picture Scaling** I see why the Storybook picture and (on mobile) the Scientist pictures are scaled to fit the width of the text columns, but I wonder if we could reduce the scaling on these so they more readily fit on a single page. I would love the Storybook picture to fit on the first page of the desktop site and for at least one scientist profile to fit on the mobile site (instead of just most of their picture).

3. **Scientists Profile Buffer** I think the buffer between rows on the Scientists page is too narrow. Could more white space be added between the rows so the scientist profile pictures don't look like they're nearly colliding with previous row?

Thank you!

ianchandlercampbell commented:
@mekline Ok! The default text size is great, so I rescind my first request!

okaycj commented on web/templatetags/web_extras.py:
```diff
@@ -114,9 +117,14 @@ def nav_link(request, url_name, text, html_classes=None, queryString=None):
-    return mark_safe(
-        f'<a class="{" ".join(html_classes)}"{aria_current} href="{url}">{_(text)}</a>'
-    )
+    if list:
+        return mark_safe(
+            f'<li class="nav-item"><a class="{" ".join(html_classes)}"{aria_current} href="{url}">{_(text)}</a></li>'
+        )
+    else:
+        return mark_safe(
+            f'<a class="{" ".join(html_classes)}"{aria_current} href="{url}">{_(text)}</a>'
+        )
```
The a-tag element is the same in both cases.  This isn't critical, but it might be better to save the a-tag to a variable and wrap it in a li-tag if list is True.

becky-gilbert replied:
Yep that is a better way. Thanks!

becky-gilbert commented:
Thanks for the feedback @ianchandlercampbell! I'm working on this now and will get back to you soon.

okaycj commented on web/templates/web/garden/home.html:
```diff
@@ -46,16 +47,16 @@
         <p>
-            <a href="#">Project GARDEN Studies</a>
+            <a href="{{ base_url }}{{ garden_url }}">Project GARDEN Studies</a>
         </p>
```
Why does this need the `base_url`?

becky-gilbert replied:
It doesn't 🤦‍♀️  Thanks for catching this - fixed!

okaycj commented on web/templates/web/garden/scientists.html:
```diff
@@ -10,4 +10,17 @@
+    <p>
+        Project GARDEN would not be possible without the work of many scientists and collaborators! Meet the team working hard to help families contribute to developmental science.
+    </p>
+    <h2 class="{% subheading_classes %}">Leadership Team</h2>
+    {% include "web/garden/_leadership_team.html" %}
+    <h2 class="{% subheading_classes %}">Grant Team Members</h2>
+    {% include "web/garden/_grant_team.html" %}
+    <h2 class="{% subheading_classes %}">Research Partners</h2>
+    {% include "web/garden/_research_partners.html" %}
+    <h2 class="{% subheading_classes %}">Other team members</h2>
+    {% include "web/garden/_other_members.html" %}
```
If `subheading_classes` is just a list of BS5 classes, then you could create a new class instead.

becky-gilbert replied:
Yes, will do!
BTW I think there are a few other template tags that are just simple lists of classes. I'll open a separate issue/PR to change those too.

okaycj commented on web/templatetags/web_extras.py:
```diff
@@ -307,3 +307,15 @@ def __init__(self, nodelist):
+    <h3 class="text-center">{name}</h3> 
+    <p>{blurb}</p>
+    </div>"""
+    return mark_safe(html)
```
I like this!

okaycj reviewed (approved):
This works!  Just a few questions.

mekline reviewed (changes requested):
This all looks great! in addition to Ian's comments, Mark has made some wording changes currently being discussed on email, I'll relay actual changes once they resolve conversation!

becky-gilbert commented:
@ianchandlercampbell @mekline 
WRT Ian's comment (2) about reducing the size of the storybook image and trying to fit all of the GARDEN home page content on a desktop/laptop screen - getting all the content on the screen is challenging because of the way the content is stacked. So as it's currently laid out, it goes beyond a single page even when the storybook and GARDEN logo images are much smaller. 

Here's what it looks like with smaller images:

![Screenshot 2023-05-18 at 4 32 15 PM](https://github.com/lookit/lookit-api/assets/9041788/7ed75d77-a6f4-4a85-a47c-eee2c19849b5)

Or if you prefer, I can reduce the length of the page by putting the GARDEN logo and page title next to one another, and/or putting the storybook image to one side of the text.

![Screenshot 2023-05-19 at 10 37 44 AM](https://github.com/lookit/lookit-api/assets/9041788/9a9448f6-1942-4e88-a644-1aae35139a5d)

Can you let me know which of these (or what combination of) options you'd prefer? I am admittedly not a graphic designer 😅 so please feel free to be picky about relative image sizing, spacing between elements etc.

ianchandlercampbell commented:
@becky-gilbert These are really good ideas, which have given me some ideas on iterations. What if we started the title element with "Welcome to" and had the Project GARDEN logo to its right? So it would still read "Welcome to Project GARDEN" while incorporating the logo. Also solves a bit of a redundancy issue that was bugging me.

For the Storybook image, what if we sized that to take half of the column width (with the 2nd and 3rd text paragraphs taking the other half). This might result in some of the paragraphs not being on the first page, but I think it would be accessible while looking visually-appealing.

becky-gilbert commented:
@ianchandlercampbell @mekline
WRT both of your comments about reducing the size of the Scientist images on mobile, here's what it looks like with 2 people per row instead of 1:

![Screenshot 2023-05-18 at 4 09 50 PM](https://github.com/lookit/lookit-api/assets/9041788/f26cda97-a181-4585-b038-c2605d938048)

And here's what it looks like with 1 person per row but with smaller images:

![Screenshot 2023-05-18 at 4 20 18 PM](https://github.com/lookit/lookit-api/assets/9041788/d17e982c-8980-4631-9a07-306264af4507)

Can you let me know which layout you prefer?

ianchandlercampbell commented:
@becky-gilbert I think I like the 1 person per row with reduced image sizes, but interested to hear what others think!

becky-gilbert commented:
> These are really good ideas, which have given me some ideas on iterations. What if we started the title element with "Welcome to" and had the Project GARDEN logo to its right? So it would still read "Welcome to Project GARDEN" while incorporating the logo. Also solves a bit of a redundancy issue that was bugging me.

> For the Storybook image, what if we sized that to take half of the column width (with the 2nd and 3rd text paragraphs taking the other half). This might result in some of the paragraphs not being on the first page, but I think it would be accessible while looking visually-appealing.

Makes sense! How's this?

![Screenshot 2023-05-19 at 11 26 15 AM](https://github.com/lookit/lookit-api/assets/9041788/6684e328-39e8-4ed7-a323-0dcf51d8fd2f)

The only thing is that this layout causes the image to go beyond the text on wide screens (screenshot above is on a 24 in monitor). As the screen narrows, the text starts to wrap past/around the bottom of the storybook image (see next screenshot).

![Screenshot 2023-05-19 at 11 27 19 AM](https://github.com/lookit/lookit-api/assets/9041788/73ee459d-5412-4191-a73c-fde38a5dedfa)

If that bothers you then I could either (a) make the image a bit smaller on wide screens:

![Screenshot 2023-05-19 at 11 10 19 AM](https://github.com/lookit/lookit-api/assets/9041788/4fe812fa-3fb2-4063-aec8-4683b84cb981)

or (b) include the 1st paragraph in the text that wraps around the image:

![Screenshot 2023-05-19 at 11 11 47 AM](https://github.com/lookit/lookit-api/assets/9041788/486ee9d7-1c08-49d3-9e98-1a66561a2304)

![Screenshot 2023-05-19 at 11 14 44 AM](https://github.com/lookit/lookit-api/assets/9041788/0c0f786a-4007-4544-a03b-914ac7149272)

becky-gilbert commented:
>3. Scientists Profile Buffer I think the buffer between rows on the Scientists page is too narrow. Could more white space be added between the rows so the scientist profile pictures don't look like they're nearly colliding with previous row?

@ianchandlercampbell how's this?

![Screenshot 2023-05-19 at 11 48 30 AM](https://github.com/lookit/lookit-api/assets/9041788/088a3651-8c0b-4bcb-a17f-22ffdf2974a9)

![Screenshot 2023-05-19 at 11 51 21 AM](https://github.com/lookit/lookit-api/assets/9041788/1bec2ace-0f7c-4433-9a13-6003e0ebd7e2)

ianchandlercampbell commented:
@becky-gilbert These Storybook Background image tweaks look great! I'm not amazing at graphic design either, so interested in what others think, but could we put a boundary so that text that starts in a right or left column stays in that column? I like the 2nd to last option with all text to the left and the Storybook picture to the right, but I don't like the text overflow after the image.

For the Scientists buffer, that looks much better!!

becky-gilbert commented:
@ianchandlercampbell thanks for the feedback! How's this?

![Screenshot 2023-05-19 at 12 02 09 PM](https://github.com/lookit/lookit-api/assets/9041788/7094d80d-184f-4e37-bd18-ac42d34baf92)

mekline commented:
(I endorse all of these changes, including the 1-per-row small image Scientists mobile page!)

ianchandlercampbell commented:
@becky-gilbert I think that's our winner! I love it.

becky-gilbert commented:
@ianchandlercampbell
Just wanted to recap where we are with this for the sake of clarity.

**GARDEN home page**
- Title is now "Welcome to" \<GARDEN project logo\>
- On desktop/laptop
  - "Welcome to" is to the left of the logo
  - Text is in the left column and storybook image in the right column. The text will stay in the column rather than wrap around the image when the page is resized.
- On mobile
  - "Welcome to" is above the logo
  - Text and storybook image are in a single column. I moved storybook image so that it appears in between the 1st and 2nd paragraphs (I can move it somewhere else if you prefer?).

**GARDEN scientists page**
- On mobile, the profiles still appear as a single column, but the image size is reduced.

**Screenshots**
![Screenshot 2023-05-22 at 4 00 19 PM](https://github.com/lookit/lookit-api/assets/9041788/b82ef4b3-6769-435f-af69-82d8ad142e82)
![Screenshot 2023-05-22 at 3 59 43 PM](https://github.com/lookit/lookit-api/assets/9041788/2af7f35c-f896-4909-942e-2ea8852f6e6d)
![Screenshot 2023-05-22 at 4 00 54 PM](https://github.com/lookit/lookit-api/assets/9041788/6153c377-4ba6-4628-862d-8bc095d918b8)

@mekline did you want me to merge this into `develop` now so that the GARDEN team can review on staging, or wait for further feedback?

becky-gilbert commented:
@okaycj you already reviewed this PR (thank you!) but I changed the staff profile template tag a bit to simplify it and adjust the image size depending on layout. Before I had the profile div col widths integers being passed in as arguments, but this made things complicated since I wanted to adjust the image size depending on the profile div size. So here's what I came up with:
- just two options for sizing/layout
- use different image sizing for these two options on mobile (full width images when profiles are in rows of 2, reduced width images when profiles are in rows of 1)
Could you please have a look at the staff template tag again and let me know what you think? Thanks!

mekline commented:
I'm meeting with the team in 6 minutes, let me check on the copy changes they had mentioned!

mekline commented:
Copy changes!

- "10-15 minutes each" --> "about 15 minutes each"

- "and families that do all 8 of our first studies will receive a $10 bonus" -> "... in Project Garden. Each study will provide a $5 gift card, and families that do all 8 of our first studies will receive a bonus $10 gift card"

- [project garden email] --> project.garden.study@gmail.com

- "brief talk on Microsoft Teams" --> "brief video call"

ianchandlercampbell commented:
@becky-gilbert Waiting on sign-off, but let me give you the text ahead of time! For the "Getting Started" section, we've revised to a 5 step process, as below. I updated the Subsite Google Doc if that's helpful for formatting, but no changes to formatting, just two new steps and calling it an "easy five-step process" (instead of a 3-step). If you don't hear otherwise from me by 10 AM ET tomorrow, you can go ahead and add this and ship it as-is.

# Getting started in Project GARDEN

 To start participating in Project GARDEN, you will follow an easy five-step process (click on each step to see specific instructions and more information):
## **(1) Create an account on Children Helping Science**

- Click on the “Sign-up” button here [link to Lookit Sign-up page], or at the top right of this website
- Follow the instructions to sign up for an account, including reading and agreeing to the Children Helping Science Privacy Statement
- Note: You will log in to your Children Helping Science account each time you and your child participate in a Project GARDEN study.

## (2) Choose a time that works for you and your child!

- Each module will be for your child (except for the Family Information Survey), so choose a time that works best for your child! Studies take about 15 minutes, but we recommend you have at least 30 minutes available for your very first study and you’re learning how it works.
- If you have more than one child, they will need to participate one at a time. Your Children Helping Science account will ask you which child is participating– make sure to choose the right child!
- We will need to see your child with you when you record your “consent video” for each study  to make sure everyone is there and ready to participate!

## (3) Get a laptop or other computer to log in to your account 

- You will need a laptop or computer with a webcam and microphone (if you have a separate webcam, this should have a microphone on it already!)
- Project GARDEN studies do not work on phones or tablets.

## (4) Visit the Project GARDEN Studies Page [https://lookit.mit.edu/studies/garden/]
- Click on the first study to get started!
- You can return to this page at any time to participate in more studies, or, after you’ve completed enough studies, re-watch some Storybook videos with Fen, our friendly fennec fox!
- At the start, only three studies will be available, but more studies will become available each month. For the first part of Project GARDEN, there will be 7 studies for your child and 1 study for you, for a total of 8 studies. More studies will be released over time!

## (5) Click on the first Project GARDEN study to start participating!
- At the start of each study, we will provide a consent form that will tell you all about the study.
- After you read the consent form, you will record a short consent video with you and your child where you say you agree to participate in the study.
- Then, you will start the study– each study will tell you exactly what to do to participate!
- Once you finish, we will check your consent video to make sure you and your child are both present and eligible for the study. All eligible families will receive a $5 Amazon.com gift card (U.S.-only) within 7 business days after participating in each study.
- Families who complete all 8 studies will receive an extra $10 Amazon.com gift card! In total, families who complete all 8 studies will receive $50 in Amazon.com gift cards (eight $5 gift cards and one extra $10 gift card).
- More studies will be available over time, with more adventures with Fen!

becky-gilbert commented:
@ianchandlercampbell great, thanks very much! I'll update now with what you have and wait until tomorrow to hear about any further changes 👍

becky-gilbert commented:
@ianchandlercampbell quick question. I notice this new bit of text at the start of the getting started section:

> ..five-step process (click on each step to see specific instructions and more information):

Does this mean that you want each step to have collapsible instructions (bullet point items) under the numbered heading? And if so, should the numbered items start out collapsed and expand (i.e. instructions become visible) when clicked?

Apologies if this is all explained in the subsite google doc and I missed it. If that's the case then please feel free to send a link here or on Slack (I checked on my google drive to see if I already have access but couldn't find it). Thanks!

becky-gilbert commented:
@ianchandlercampbell @mekline I will merge this now so that you can see everything on staging. Here's a recap of the last set of changes I made since the minor copy changes from [Melissa's comment above](https://github.com/lookit/lookit-api/pull/1221#issuecomment-1559690902).

- Changed the Getting Started content from 3-step to 5-step process per [Ian's comment above](https://github.com/lookit/lookit-api/pull/1221#issuecomment-1560138552)
- Converted the Getting Started section (on the participate page) and FAQ section (on the about page) from static content into accordions (click on heading to see text appear below)
- Fixed link to the GARDEN studies page in the step 4 heading in the Getting Started section (broke because of the switch to accordion)
- Added the GARDEN home page link onto the CHS home page.

@ianchandlercampbell could you please double check the Getting Started section in particular once this is on staging, just to make sure all of the content changes are correct? (I haven't posted screenshots of all of the sections here). Thanks!

**Getting Started section: desktop/laptop**

![Screenshot 2023-05-24 at 10 30 39 AM](https://github.com/lookit/lookit-api/assets/9041788/938e8aa2-9d0d-4f12-a676-101e8daa78a1)
![Screenshot 2023-05-24 at 10 30 50 AM](https://github.com/lookit/lookit-api/assets/9041788/b755e2b7-1c43-4dbb-b945-5ab6b24fce4b)
![Screenshot 2023-05-24 at 10 31 32 AM](https://github.com/lookit/lookit-api/assets/9041788/5e1d5dce-e3d6-4318-920c-cb4af417e108)

**Getting Started section: mobile**

![Screenshot 2023-05-24 at 12 14 39 PM](https://github.com/lookit/lookit-api/assets/9041788/1a0354e5-9ef2-427c-a435-0e5210bab597)

**FAQ section: desktop/laptop**

![Screenshot 2023-05-24 at 12 15 13 PM](https://github.com/lookit/lookit-api/assets/9041788/67562cc9-46d2-4ff1-9e77-452df479653a)
![Screenshot 2023-05-24 at 12 15 26 PM](https://github.com/lookit/lookit-api/assets/9041788/96effa01-8c31-4fff-b08a-9a2a31abafe8)

**FAQ section: mobile**

![Screenshot 2023-05-24 at 12 14 55 PM](https://github.com/lookit/lookit-api/assets/9041788/5b424b3d-e0e8-4f67-be2b-9c2579360105)

**GARDEN link on CHS home page**

![Screenshot 2023-05-24 at 12 12 26 PM](https://github.com/lookit/lookit-api/assets/9041788/18073fe6-bf43-4d30-8719-ec9f60ea6cba)
